### PR TITLE
Tmz 3.2.0 refactor number sequences issue 1822

### PR DESCRIPTION
**Pull request by t-m-z**
refactor
add async
add button

there seems still to be a problem with Bell

### CODE REVIEW

File: lib/tools/science_and_technology/number_sequences/_common/widget/numbersequences_nthnumber.dart [hunk 1]
```diff
@@ -46,12 +55,51 @@ class _NumberSequenceNthNumberState extends State<NumberSequenceNthNumber> {
...
+          });
+        }),
+        _currentOutput,
+        //_buildOutput()
       ],
     );
   }
 
-  Widget _buildOutput() {
+  /*Widget _buildOutput() {
```
Comment (Mike-3) [inline lib/tools/science_and_technology/number_sequences/_common/widget/numbersequences_nthnumber.dart:69]: remove unused code

File: lib/tools/science_and_technology/number_sequences/_common/logic/number_sequence_range.dart [hunk 2]
```diff
@@ -1,22 +1,52 @@
 part of 'package:gc_wizard/tools/science_and_technology/number_sequences/_common/logic/number_sequence.dart';
 
-List<BigInt> numberSequencesGetNumbersInRange(NumberSequencesMode sequence, int? start, int? stop) {
-  if (start == null || stop == null) return [BigInt.from(-1)];
+
+/*
```
Comment (Mike-3) [inline lib/tools/science_and_technology/number_sequences/_common/logic/number_sequence_range.dart:4]: remove unused code

**Review by Mike-3 — CHANGES_REQUESTED** [review]

File: lib/tools/science_and_technology/number_sequences/_common/logic/number_sequence_range.dart (hunk 2 above)
Comment (t-m-z) [reply lib/tools/science_and_technology/number_sequences/_common/logic/number_sequence_range.dart]: done

File: lib/tools/science_and_technology/number_sequences/_common/widget/numbersequences_nthnumber.dart (hunk 1 above)
Comment (t-m-z) [reply lib/tools/science_and_technology/number_sequences/_common/widget/numbersequences_nthnumber.dart]: done

**Pull request merged**